### ISSUE TITLE
Does not work with Gradle 7.0:  missing an input or output annotation

### ISSUE THREAD

**Issue by henrik242** [dorongold/gradle-task-tree#43]
The fix for this is probably akin to that of https://github.com/psxpaul/gradle-execfork-plugin/pull/38 which was reported at https://github.com/psxpaul/gradle-execfork-plugin/issues/36
```
* What went wrong:
Some problems were found with the configuration of task ':frontpage-web:taskTree' (type 'TaskTreeTaskNew').
  - Type 'TaskTreeTaskNew' property 'noRepeat' is missing an input or output annotation.
    
    Reason: A property without annotation isn't considered during up-to-date checking.
    
    Possible solutions:
      1. Add an input or output annotation.
      2. Mark it as @Internal.
    
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#missing_annotation for more details about this problem.
  - Type 'TaskTreeTaskNew' property 'taskDepth' is missing an input or output annotation.
    
    Reason: A property without annotation isn't considered during up-to-date checking.
    
    Possible solutions:
      1. Add an input or output annotation.
      2. Mark it as @Internal.
    
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#missing_annotation for more details about this problem.
```

**Comment by baron1405** [comment]
Hit this today as well. The Task Tree plugin is especially important with Gradle 7 being super sensitive to task input specification. I was going to use it to debug one such issue but hit this problem, which itself is due to Gradle 7's super sensitivity to task annotations.

**Comment by sarnobat** [comment]
I can't get it to work with 4.8.1 either.

**Comment by henrik242** [comment]
> I can't get it to work with 4.8.1 either.

I guess that's a completely different problem, unless you have the exact same exception? It works flawlessly with Gradle 6.x

**Comment by xenomachina** [comment]
@henrik242 It's been [broken since 6.8](https://github.com/dorongold/gradle-task-tree/issues/40), but the 7.0 issue appears to have a different root cause.

**Comment by henrik242** [comment]
@xenomachina Oh, it worked for me in 6.8 :)

**Comment by henrik242** [comment]
@xenomachina I've switched to https://gitlab.com/barfuin/gradle-taskinfo now, it does all I needed task-tree for.

**Comment by hkhc** [comment]
> @xenomachina I've switched to https://gitlab.com/barfuin/gradle-taskinfo now, it does all I needed task-tree for.

Oh, you save my day! I like the feature to save the dependency tree as JSON, which is exactly my needs to test my Gradle plugin.

**Comment by dorongold** [comment]
Fixed by fb19b847e2a6c419092c2231cc1839377bfb675b.
